app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="row">
+  <div class="col-md-6 offset-md-3">
+  <h2>Sign up to our blog</h2>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="form-group mt-2">
+    <%= f.label :Name %><br />
+    <%= f.text_field :Name, autofocus: true, autocomplete: "Full Name", required: true, class: 'form-control' %>
+  </div>
+
+  <div class="form-group mt-2">
+    <%= f.label :Bio %><br />
+    <%= f.text_area :Bio, size: "50x5", class: 'form-control', required: true %>
+  </div>
+
+  <div class="form-group mt-2">
+    <%= f.label :Photo %><br />
+    <%= f.url_field :Photo, class: 'form-control', required: true %>
+  </div>
```

Comment:
- [Optional] I don't think these two fields are necessary when signing up. It really makes the process of signing up difficult and the visitor may field like to abandon the process. I think you should have made the photo to be uploadable instead of asking for a URL with some constraint. Let's consider the best user experience when developing an application.